docs/schemas.md

```diff
@@ -267,6 +268,28 @@ A Content tooltip block is an in-line block which allows content developers to a
 <span class="os-raise-ib-tooltip" data-store="storename" data-schema-version="1.0"><!-- INSERT TERM HERE --></span>
 ```
 
+### Desmos graphing calculator block
+
+#### Description
+
+The Desmos graphing calculator block displays a graph and an expressions panel. Add expressions to the graph by passing a JSON array into the optional `data-equations` attribute, where each element represents a new equation or point that will be added to the expressions panel. The `data-bottom`, `data-top`, `data-left`, and `data-right` attributes take number values and define where the graph viewport sits. If only the graph is needed, the expressions panel can be removed by passing the optional `data-disable-expressions` attribute. The size of the Desmos interactive can be set by passing an integer value into the `data-width` and `data-height` attributes. 
+
+#### Schema definition
+
+The following template provides an example with a Desmos graphing calculator block. 
+
+```html
+<div class="os-raise-ib-desmos-gc" data-wait-for-event="eventnameY"  data-height="500" data-width="600"  data-equations='["(1,2)","x=5"]' data-disable-expressions data-top="50" data-bottom="-50" data-left="-50" data-right="50" data-schema-version="1.0">
+</div>
+```
+
+Notes on schema:
+
+* The `data-wait-for-event` attribute is optional and where specified should correspond to a `data-fire-event` from another Interactive Block on the same page (e.g. as part of a Content Template)
+* The height and width attributes are optional and the default is 400 by 400 pixels.
+* The default values for the optional `data-bottom`, `data-top`, `data-left`, and `data-right` attributes are 10 for the top and right and -10 for bottom and left. 
+* if you just want to render a graphing calculator with nothing pre-graphed you can do so with `<div class="os-raise-ib-desmos-gc"></div>`
```

Comment:
Minor: Let's add the `data-schema-version="1.0"` in the example on this line too.